src/scss/_palette.scss

```diff
@@ -54,6 +54,8 @@ $o-colors-palette: map-merge((
 	'org-b2c':               #4e96eb,
 	'org-b2c-dark':          #3a70af,
 	'org-b2c-light':         #99c6fb,
+
+	'internal-slate':         #262a33,
```

Comment:
Some documentation, maybe just a comment, to explain why `internal-slate` is needed would be great. I.e. tint via mix vs. saturation change.